docs/documentation/server_development/topics/providers.adoc

```diff
@@ -288,6 +289,21 @@ function authenticate(context) {
 }
 ----
 
+===== Where to add script authenticator
+
+One of the possible use of script authenticator is to do some checks at the end of the authentication. Note that if you want
+your script authenticator to be always triggered (even for instance during SSO re-authentication with the identity cookie), it may be needed to add it as REQUIRED at the end
+of the authentication flow and encapsulate the existing authenticators into separate REQUIRED authenticator. This is due the REQUIRED and ALTERNATIVE executions
```

Comment:
```suggestion
of the authentication flow and encapsulate the existing authenticators into a separate REQUIRED authenticator. This need is because the REQUIRED and ALTERNATIVE executions
```